backend/app/routes/teamMember/index.js

```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router({ mergeParams: true });
+const addTeam = require('./addTeam');
+
+const { FetchUser } = require('../../../helpers/middlewares/auth');
+const multer = require('multer');
+
+const upload = multer({ dest: 'uploads/teamMembersProfile' });
```

Comment:
Can we please think of other way without introducing a new folder `upload` for just storing the images?